examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp

```diff
@@ -57,10 +83,25 @@ void FabricSyncAddBridgeCommand::OnCommissioningComplete(chip::NodeId deviceId,
                         ChipLogValueX64(mBridgeNodeId));
 
         char command[kMaxCommandSize];
-        snprintf(command, sizeof(command), "descriptor subscribe parts-list %d %d %ld %d", kSubscribeMinInterval,
+
+        // Prepare and push the descriptor subscribe command
+        snprintf(command, sizeof(command), "descriptor subscribe parts-list %d %d %" PRIu64 " %d", kSubscribeMinInterval,
```

Comment:
separate objects here as well: why are we using commands via strings? This would not be integratable to some other applications.

Reply:
https://github.com/project-chip/connectedhomeip/issues/34910